tests/dummy/app/templates/index.hbs

```diff
@@ -45,6 +45,11 @@
   <p>You can find out more information on the CSP addon page <a href="https://github.com/rwjblue/ember-cli-content-security-policy#ember-cli-content-security-policy">here</a>.</p>
   <p>We also need to allow `data:` in `img-src` because of a <a href="https://github.com/miguelcobain/ember-paper/blob/master/app%2Fstyles%2Fpaper-button.scss">current hack</a> in paper-button.</p>
 
+  <p>
+    Finally, the examples contain a `transition-to` helper that is not installed by default. To use this helper run the following:
```

Comment:
I'm thinking this would be better added to the side-nav, even though it relates to installation. Only someone who both uses the side-nav and is copying the template would need this info. That way we don't overwhelm someone who is just getting started. Also, is there a reason why `ember install ember-route-action` wouldn't work? It also runs blueprints and I think just wraps `npm install --save-dev`, no? Last, if there is is a reason, I'd prefer the longer form of the option for clarity. Thanks for contributing!!